.evergreen/config_generator/components/cse/darwinssl.py

```diff
@@ -0,0 +1,75 @@
+from shrub.v3.evg_build_variant import BuildVariant
+from shrub.v3.evg_task import EvgTaskRef
+
+from config_generator.etc.compile import generate_compile_tasks
+
+from config_generator.etc.cse.compile import CompileCommon
+from config_generator.etc.cse.test import generate_test_tasks
+
+
+SSL = 'darwinssl'
+TAG = f'cse-matrix-{SSL}'
+
+
+# pylint: disable=line-too-long
+# fmt: off
+COMPILE_MATRIX = [
+    ('macos-1015', 'clang', None, ['cyrus']),
```

Comment:
It seems like [this task](https://spruce.mongodb.com/task/mongo_c_driver_cse_matrix_darwinssl_cse_sasl_cyrus_darwinssl_macos_1015_clang_compile_patch_04d975b2e140e9a40859f4a44e533959bdaffa37_63eab7593e8e863afae925c0_23_02_13_22_19_07/logs?execution=0) may be blocked by macos-1015 [not yet(?) providing](https://parsley.mongodb.com/evergreen/mongo_c_driver_cse_matrix_darwinssl_cse_sasl_cyrus_darwinssl_macos_1015_clang_compile_patch_04d975b2e140e9a40859f4a44e533959bdaffa37_63eab7593e8e863afae925c0_23_02_13_22_19_07/0/task?bookmarks=0,881&shareLine=557) the CoreFoundation library (aka Secure Transport) required by both libmongocrypt and libmongoc when Client Side Encryption is enabled.

Reply:
I forgot I found the same problem in libmongocrypt before and filed MONGOCRYPT-443. I suggest commenting out cse tasks on macOS until MONGOCRYPT-443 is resolved.

Reply:
Rather than commenting out the tasks, I added `disable: true` to the tasks instead and renamed `EvgTaskWithRunOn` to simply `Task` accordingly. Per [docs](https://github.com/evergreen-ci/evergreen/wiki/Project-Configuration-Files#limiting-when-a-task-will-run):

> This behaves similarly to commenting out the task but will not trigger any validation errors.